app/assets/stylesheets/repository/repository_table.scss

```diff
@@ -34,14 +34,36 @@
   // Assigned
 
   .assigned-column {
+    position: relative;
+
     .repository-row-edit-icon {
       cursor: pointer;
       display: none;
       width: 30px;
     }
 
-    .circle-icon {
+    .assign-counter-container {
+      border-radius: 2px;
+      cursor: pointer;
+      display: inline-block;
+      height: calc(100% - 2px);
       margin-left: 30px;
+      padding-top: 7px;
+      position: absolute;
+      top: 1px;
+      width: calc(100% - 38px);
+
+      .assign-counter {
+        margin-left: 5px;
+
+        &.has-assigned {
+          color: $brand-primary;
+        }
+      }
+
+      &:hover {
+        background-color: $color-silver-chalice;
```

Comment:
Must be `$color-alto` - https://app.zeplin.io/project/5cb50449f9317b134dcee5e3/screen/5dc38bb583c70b83631e9239